app/src/main/res/values/attrs.xml

```diff
@@ -15,6 +15,7 @@
     <attr name="tabRing" format="reference" />
     <attr name="snackbar" format="reference" />
     <attr name="foundation" format="reference" />
+    <attr name="foundationContrast" format="reference" />
```

Comment:
I am also wondering if we need this entry in the `attrs.xml` since I usually see another reference in `values/styles.xml`. Technically, this isn't being used at all atm.

Reply:
Yeah, sure! I was adding this until I realised we do not need it for this use case, as we don't have top sites on private mode.